qcodes/instrument_drivers/AlazarTech/ATS9360.py

```diff
@@ -11,12 +11,10 @@ class AlazarTech_ATS9360(AlazarTech_ATS):
     """
     This class is the driver for the ATS9360 board
     it inherits from the ATS base class
-
     TODO(nataliejpg):
         -  add clock source options and sample rate options
            (problem being that byte_to_value_dict of
            sample_rate relies on value of clock_source)
-
```

Comment:
Please revert all changes of this file. Removing these blank lines breaks formatting of this docstring